utils/types/reporting_types.go

```diff
@@ -113,8 +113,11 @@ type EDErrorDetails struct {
 	ErrorCode    string `json:"errorCode"`
 	ErrorMessage string `json:"errorMessage"`
 	// TODO: need to check with team if this makes sense ?
-	EventType string `json:"-"`
-	Count     int    `json:"count"`
+	EventType      string `json:"-"`
+	EventName      string `json:"-"`
```

Comment:
```suggestion
	EventType      string `json:"eventType"`
	EventName      string `json:"eventName"`
```
As these also should be stored in `errors` table right ?!